text/0055-nested-multi-sigs.md

```diff
@@ -0,0 +1,101 @@
+- Feature Name: nested_multi_sig
```

Comment:
:wrench: Looking at some other MCIPs the feature name is human text (versus snakecase code, not a good term)
```suggestion
- Feature Name: Nested Minting Multi-Signatures
```
Not sold on my wording but thought "Minting" should be in there for the block version table